src/plugins/elasticsearch/lib/create_proxy.js

```diff
@@ -16,10 +16,11 @@ function createProxy(server, method, route, config) {
     handler: {
       proxy: {
         mapUri: mapUri(server),
-        passThrough: true,
         agent: createAgent(server),
-        xforward: true,
```

Comment:
I think we probably want to keep `xforward` enabled.  In this case, this is a header specifically injected by Kibana itself rather than a header sent by the user's browser.  This type of header only makes sense in the context of the direct proxy.

What do you think?